LICENSE

```diff
@@ -1,19 +1,18 @@
-Some files are licensed under GPLv3.  These files have the text
+Some files are licensed under GPL-3.0.  These files have the text
 
-    SPDX-License-Identifier: GPLv3
+    SPDX-License-Identifier: GPL-3.0
 
 in the file header.
 
 All other files, unless noted, are licensed under the MIT license.
 
-= GPLv3+ license text
+= GPL-3.0 license text
```

Comment:
You don't have to use the SPDX identifier here

Reply:
But you should not use the `+` which AIUI implies the "or later" part.